server/services/forms/schemas/add-standard-line.js

```diff
@@ -14,6 +14,9 @@ module.exports = options => Form()
         Component('date', 'expiry_date')
             .withValidator('required', 'Expiration date is required')
             .withValidator('isValidDate', 'Expiration date must be a valid date')
+            .withValidator('isValidDay', 'Date must contain a valid day')
+            .withValidator('isValidMonth', 'Date must contain a valid month')
+            .withValidator('isYearWithinRange', 'Date must be before ' + (new Date().getFullYear() + 120))
```

Comment:
Magic number?

Reply:
Good point, removed from here and several other places